good-enough-practices-for-scientific-computing.tex

```diff
@@ -264,13 +267,19 @@ \subsection*{Acknowledgments}
   data by accident or to hand edit it in a moment of weakness.
 
   Some data will be impractical to manage in this way. For example, you should avoid
-  making local copies of large, stable repositories.
+  making local copies of large, stable databases.
   In that case, record the exact procedure used to obtain the raw data, as well
   as any other pertinent information, such as an official version number or the date of download. 
   
-  Ensure that raw data is backed up in more than one location. For large data sets, 
+  \practice{1b}{Ensure that raw data is backed up in more than one location.} If external hard drives are used, 
+  ideally they are stored off-site of the original location. Universities often have their own data
+  storage solutions, so it is worthwhile to consult with your local Information Technology (IT) or library groups.
+  Alternatively cloud computing resources, like Amazon
```

Comment:
Amazon, Microsoft Azure or Google Cloud (we usually mention multiple tools, see also #181)